frontend/src/app/app.module.ts

```diff
@@ -27,8 +32,11 @@ import {RestaurantsService} from "./shared/restaurants.service";
     MatButtonModule,
     MatCardModule,
     MatToolbarModule,
-    HttpClientModule
+    HttpClientModule,
+    NgQrScannerModule,
+    MatProgressSpinnerModule
   ],
+  schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
   providers: [RestaurantsService],
```

Comment:
why you need  to add CUSTOM_ELEMENTS_SCHEMA !

Reply:
as seen with you the other time, it's useful when you use a component whose tag name contains hyphen, otherwise an error occurs at compile time